BUILD.md

```diff
@@ -24,3 +24,17 @@ tito tag
 ```
 tito build --rpm
 ```
+
+## Build openshift-ansible container image
```

Comment:
> I believe that moving the build instructions into BUILD.md makes sense, so I'd vote for updating the title/description in that file and link to it from the container image readme as you mention below.

:+1: 

Just note that there are other references pointing to BUILD.md, e.g. I know one in README.md, so those might need some adjustment too.